src/components/ExpandCollapseSection.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Icon } from 'patternfly-react'
+
+const SECTION_OPENED_ICON = {
+  false: {
+    type: 'fa',
+    name: 'angle-right',
+  },
+  true: {
+    type: 'fa',
+    name: 'angle-down',
+  },
+}
+
+/*
+ * React implementation of https://www.patternfly.org/pattern-library/forms-and-controls/expand-collapse-section/
+ */
+class ExpandCollapseSection extends React.Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      opened: props.opened || false,
+    }
+
+    this.handleToggleExpand = this.handleToggleExpand.bind(this)
+  }
+
+  handleToggleExpand (e) {
+    if (e && e.preventDefault) e.preventDefault()
```

Comment:
And as I already asked in previous PR, is this `if (e && e.preventDefault)` check really need?

Reply:
Here the check isn't needed, but the `e.preventDefault()` is.